src/Http/ResponseEmitter.php

```diff
@@ -42,12 +42,33 @@
 class ResponseEmitter implements EmitterInterface
 {
     /**
-     * @inheritDoc
+     * Maximum output buffering size for each iteration.
+     *
+     * @var int
+     */
+    protected $maxBufferLength;
+
+    /**
+     * Constructor
+     *
+     * @param int $maxBufferLength Maximum output buffering size for each iteration.
+     */
+    public function __construct(int $maxBufferLength = 8192)
+    {
+        $this->maxBufferLength = $maxBufferLength;
+    }
+
+    /**
+     * Emit a response.
+     *
+     * Emits a response, including status line, headers, and the message body,
+     * according to the environment.
+     *
+     * @param \Psr\Http\Message\ResponseInterface $response The response to emit.
+     * @return bool
      */
-    public function emit(ResponseInterface $response, $maxBufferLength = 8192): bool
+    public function emit(ResponseInterface $response): bool
```

Comment:
Although I doubt anyone is using it we should document this change in the migration guide.

Reply:
Will do, this was necessary as the interface changed and the method no longer has the 2nd parameter.